pay/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+
+## 1.2.0 (2023-02-02)
+* Updated `pay_android` to `1.1.0` which includes the latest `flutter_svg` package.
```

Comment:
Let's use present simple to be consistent with the rest of the log. Suggestion:
* Update `pay_android` to `1.1.0`, which includes the latest version of the `flutter_svg` package.

Reply:
Fixed in 5bfa1a6389dd13c70f4b4982db7f9d02ffa176b3